src/js/calculate.js

```diff
@@ -208,10 +243,18 @@ export default {
 
         const newModelValue = props.dataNodesObj.getVal();
 
+        // It's possible there's a better way to do this, but this works around
+        // an error where `odk:setgeopoint` inside a model can't find an associated
+        // `.question` in the DOM.
+        if ( !control || this.form.input.getWrapNode( control ) == null ) {
```

Comment:
I can't find it now but I feel like I remember you mentioning you thought there could be a deeper bug or error state that needs handing here even for `setvalue`. Does that sound right?

Reply:
I think you may be thinking of the `repeat` async issue, but looking at this again actually helped me discover that I'd missed a corresponding change in `input.js` that makes this check unnecessary. Thanks!